pool/src/pool.rs

```diff
@@ -397,7 +394,20 @@ impl Pool {
 				}
 			}
 		}
+
+		// Now sort them by depth (ascending) to maintain dependency ordering.
+		// And then sort them (within each depth) by fee_to_weight descending.
+		// Txs with no dependencies will be toward the start of the vec.
+		// Txs with a big chain of dependencies will be toward the end of the vec.
+		// Will satisfying the "depth" ordering we will then sort txs such that high
```

Comment:
Small typo `While satisfying the "depth" ordering we will then sort txs such that high` ?